src/Components/Publishing/Display/DisplayPanel.tsx

```diff
@@ -47,70 +60,89 @@ export class DisplayPanel extends React.Component<DisplayPanelProps, null> {
     unit_layout: "panel"
   }))
   openLink(e) {
+    const { link } = this.props.unit
     e.preventDefault()
-    window.open(this.props.unit.link.url, '_blank')
+    if (link) {
+      window.open(link.url, '_blank')
+    }
   }
 
-  onMouseEnter = () => {
+  @track(once((props) => ({
```

Comment:
`once` is probably not necessary here as I believe we'll actually want to track multiple plays.